smack-extensions/src/main/java/org/jivesoftware/smackx/bob/provider/BoBExtensionProvider.java

```diff
@@ -0,0 +1,63 @@
+/**
+ *
+ * Copyright 2018
```

Comment:
A copyright statement without an indication who owns the copyright is worthless. If you don't want to have the copyright, assign it to the "Ignite Realtime Foundation".